R/draws-index.R

```diff
@@ -173,6 +175,31 @@ set_variables <- function(x, variables, ...) {
   return(x)
 }
 
+#' @rdname set_variables
+#' @param old_variables (character) Variable names to repair. Should be variable
```

Comment:
For now I went with `old_variables` as the argument name, but happy to change this based on feedback.